upgrade/breaking-changes/8-0-0.md

```diff
@@ -0,0 +1,22 @@
+---
+title: 8.0.0
+description: Handle the changes in the 8.0.0 release version of the Telerik UI for .NET MAUI components.
+page_title: Breaking Changes in 8.0.0
+slug: changes-in-8-0-0
+position: 1
+---
+
+# Breaking Changes in Version 8.0.0
+
+This article lists the breaking changes introduced with the Telerik UI for .NET MAUI 8.0.0 release version.
+
+
+## Map
+
+* The Map component is moved to the `Telerik.Maui.Controls` namespace.
+* The `ShapefileLayer` is changed to `MapShapefileLayer`.
+* The `GetBestView` method is of type `Telerik.Maui.Controls.ShapefileReader.LocationRect`.
```

Comment:
To be exhaustive, you may want to mention the previous type. This comment applies to all methods whose type got changed.